### PR TITLE
Update book for Turbine

### PR DESCRIPTION
**Pull request by aeyakovenko**
#### Problem

Book doesn't name the data fanout technique by name.

#### Summary of Changes

Update book.

Fixes #

### CODE REVIEW

File: book/src/turbine-block-propagation.md [hunk 1]
```diff
@@ -26,6 +26,14 @@ make up layer 0. These will automatically be the highest stake holders, allowing
 the heaviest votes to come back to the leader first. Layer-0 and lower-layer
 nodes use the same logic to find their neighbors and next layer peers.
 
+To reduce the possiblity of attack vectors, each blob is transmitted over a
+random tree of neighboorhoods.  Each node uses the same set of nodes represeting
+the cluster.  A random tree is generated for each blob using randomness derived
+from the blob itself.  Since the random seed is not known in advance, attacks
+that try to eclipse neighboorhoods from certain leaders or blocks become very
+difficult, and should require almost complete control of the stake in the
```
Comment (garious) [inline book/src/turbine-block-propagation.md]: I don't understand this solution. Why wouldn't an attacker try to eclipse *all* the neighborhoods? It's not like there's millions of them.

File: book/src/turbine-block-propagation.md [hunk 2]
```diff
@@ -26,6 +26,14 @@ make up layer 0. These will automatically be the highest stake holders, allowing
 the heaviest votes to come back to the leader first. Layer-0 and lower-layer
 nodes use the same logic to find their neighbors and next layer peers.
 
+To reduce the possiblity of attack vectors, each blob is transmitted over a
+random tree of neighboorhoods.  Each node uses the same set of nodes represeting
+the cluster.  A random tree is generated for each blob using randomness derived
+from the blob itself.  Since the random seed is not known in advance, attacks
+that try to eclipse neighboorhoods from certain leaders or blocks become very
```
Comment (garious) [inline book/src/turbine-block-propagation.md]: `:set spell`

File: book/src/turbine-block-propagation.md (hunk 1 above)
Comment (aeyakovenko) [reply book/src/turbine-block-propagation.md]: how would they do that without all the stake?

**Comment by garious** [comment]
The file rename will change the http link, so please merge as soon as your happy with it, so that we can start sharing the new link.

**Pull request merged**